addons/website_product_comparison/static/src/xml/comparison.xml

```diff
@@ -0,0 +1,48 @@
+<templates id="compare_products" xml:space="preserve">
+
+    <t t-name="product_feature_template">
+        <div class="o_product_feature_panel hidden-xs css_editable_mode_hidden" style="display:none" />
+    </t>
+
+    <t t-name="product_comparison_template">
+        <span class="o_product_panel" id="comparelist">
+            <span class="o_product_panel_header">
+                <span class="o_product_icon"><i class="fa fa-exchange" /></span>
+                <span class="o_product_text">Compare</span>
+                <span class="o_product_circle label label-primary">0</span>
+            </span>
+            <span class="o_product_panel_content">
+                <div class="o_product_panel_empty text-left">
+                    <h4>Click on <i class="fa fa-exchange" /> to add items to your comparison list</h4>
```

Comment:
`<i />` is [invalid HTML](https://www.w3.org/TR/html5/syntax.html#start-tags), use `<i></i>` instead (yes I know we do it at many other places but it is still wrong, same for `<span />`)